CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 
 ### Features
 - Add NHID support in `ScanSimpleIdScreen`
+- Add NHID in `PatientEntryScreen`
```

Comment:
Might not be that important to change.. But, shouldn't the changelog update be more like `Add support for NHID in `PatientEntryScreen``? As we have already been adding nhid in patient entry screen before but it was in the bp passport section?